docs/connecting-to-a-lit-network/lit-blockchains/chronicle.md

```diff
@@ -0,0 +1,56 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+import AddRollupButton from "@site/src/components/AddRollupButton";
+
```

Comment:
can we add an info callout box at the top of this page saying that this rollup has been deprecated and that folks should make sure to use Vesuvius?